src/DataDog/Az.DataDog.psd1

```diff
@@ -0,0 +1,141 @@
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.3.0'; })
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = './bin/Az.DataDog.private.dll'
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+FormatsToProcess = './Az.DataDog.format.ps1xml'
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+# NestedModules = @()
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = 'Get-AzDataDogMarketplaceAgreement', 'Get-AzDataDogMonitor', 
+               'Get-AzDataDogMonitorApiKey', 'Get-AzDataDogMonitorDefaultKey', 
+               'Get-AzDataDogMonitorHost', 'Get-AzDataDogMonitorLinkedResource', 
+               'Get-AzDataDogMonitorMonitoredResource', 
+               'Get-AzDataDogSingleSignOnConfiguration', 'Get-AzDataDogTagRule', 
+               'New-AzDataDogFilteringTagObject', 
+               'New-AzDataDogMarketplaceAgreement', 'New-AzDataDogMonitor', 
+               'New-AzDataDogSingleSignOnConfiguration', 'New-AzDataDogTagRule', 
+               'Remove-AzDataDogMonitor', 'Set-AzDataDogMonitorDefaultKey', 
+               'Update-AzDataDogMonitor', 'Update-AzDataDogMonitorSetPasswordLink'
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @()
+
+# Variables to export from this module
+# VariablesToExport = @()
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = '*'
+
+# DSC resources to export from this module
+# DscResourcesToExport = @()
+
+# List of all modules packaged with this module
+# ModuleList = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','ResourceManager','ARM','PSModule','DataDog'
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://aka.ms/azps-license'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/Azure/azure-powershell'
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
+
+        # ReleaseNotes of this module
+        # ReleaseNotes = ''
```

Comment:
Should add release note.

Reply:
Release notes should leave empty. But ChangeLog.md is incorrect for the description should be under upcoming release. Will fix it in another.